src/test/groovy/grails/plugin/sentry/SanityIntegrationSpec.groovy

```diff
@@ -4,43 +4,38 @@ import ch.qos.logback.classic.Logger
 import com.stehno.ersatz.ErsatzServer
 import grails.testing.mixin.integration.Integration
 import io.sentry.Sentry
-import io.sentry.SentryClient
-import io.sentry.SentryClientFactory
+import io.sentry.logback.SentryAppender
 import io.sentry.servlet.SentryServletRequestListener
 import org.slf4j.LoggerFactory
 import spock.lang.Specification
-import spock.util.mop.ConfineMetaClassChanges
 
 @Integration
 class SanityIntegrationSpec extends Specification {
 
-    GrailsLogbackSentryAppender sentryAppender
-    SentryClient sentryClient
-    SentryClientFactory sentryFactory
-    SentryClientFactoryProvider sentryClientFactoryProvider
+    SentryAppender sentryAppender
+    GrailsEventProcessor grailsEventProcessor
     SentryServletRequestListener sentryServletRequestListener
 
-    @ConfineMetaClassChanges(GrailsLogbackSentryAppender)
     def "everything works"() {
         expect: "if everything is ok sentry then beans are injected"
             sentryAppender
-            sentryClient
-            sentryFactory
-            sentryClientFactoryProvider
+            grailsEventProcessor
             sentryServletRequestListener
         when: "mock http server is started"
             ErsatzServer server = new ErsatzServer().expectations {
-                post("/api/123/store/") {
+                post("/api/123/envelope/") {
                     called 1
                     responder {
                         code 200
                     }
                 }
             }
             server.start()
+            println "Ersatz server started at localhost:${server.httpPort}"
```

Comment:
prinltn ?

Reply:
Yeah, logging is not actually displayed during test, this is still visible only with 
```
test {
    testLogging {
        showStandardStreams = true
    }
}
```